lib/features/push_notification/data/model/web_socket_echo_request.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:jmap_dart_client/jmap/core/capability/capability_identifier.dart';
+
+class WebSocketEchoRequest {
```

Comment:
Should create abstract class `WebSocketRequest`

Reply:
Done